### ISSUE TITLE
Don't delete bundles before build

### ISSUE THREAD

**Issue by hlfcoding** [jhnstn/bundbi#5]
This causes problems with Git clients that see the deletions as a potentially huge diff, which causes memory and processing problems of varying degrees for clients.

**Comment by jhnstn** [comment]
@hlfcoding bundbi does not delete the bundle. It _does_ open a file stream for browserify to write to. Between the time that the stream is open and browserify is creating the bundle, the file size does drop to 0. This is not unique to bundbi. Browserify's cli does the same thing.

I used [this script](https://gist.github.com/jhnstn/4ae6b7e1916e3fb7be36) to time how long the file size drops to zero. Both bundbi and browserify are within the same order of magnitude of time where the bundle is empty. I did find a case where bundbi is a bit slower.(~30 sec vs ~20 sec)  I'm looking into that.

I suppose bundbi could use some temp file to stream and then do one swap at the end. However performance parity with browserify should suffice.

**Comment by hlfcoding** [comment]
@jhnstn I think that's what we did before, and I'm only reporting what I saw from the git client. Constantly updating the file probably creates an overwhelming number of notifications for any watchers, which explains the hanging of other apps and why the file looks deleted until it's there (or the client crashes and restarts). There should definitely be (at least an option for) writing to a temp file.
